pkg/apitype/policy.go

```diff
@@ -64,12 +68,31 @@ type Policy struct {
 	DisplayName string `json:"displayName"`
 
 	// Description is used to provide more context about the purpose of the policy.
-	Description      string           `json:"description"`
+	Description string `json:"description"`
+
+	// This enforcement level will be deprecated in favor of enforcement
+	// level specified by the default configuration.
 	EnforcementLevel EnforcementLevel `json:"enforcementLevel"`
 
 	// Message is the message that will be displayed to end users when they violate
 	// this policy.
 	Message string `json:"message"`
+
+	// DefaultConfiguration is used to specify the default configuration
+	// for the Policy.
+	DefaultConfiguration PolicyConfiguration `json:"defaultConfiguration"`
+}
+
+// PolicyConfiguration defines the configurable values of a
+// specific Policy.
+type PolicyConfiguration struct {
+	// If true, the policy is not run. Defaults to false.
+	Disabled         bool             `json:"disabled"`
+	EnforcementLevel EnforcementLevel `json:"enforcementLevel"`
```

Comment:
I can see us having these be implicit or specialized configuration variables, that we don't need to explicitly put into the API type. For example, they could just be from configuration values "pulumi:disabled" and "pulumi:enforcementLevel".

But for convenience, if we expect `Disabled` and `EnforcementLevel` to be set for _every single policy rule_, then having them as fields on `PolicyConfiguration` makes more sense.

Reply:
I made this `Disabled` was that it will default to false, therefore if not set explicitly it will automatically be enabled.